README.md

```diff
@@ -1 +1,235 @@
-# IoT-STH
+#<a id="section0"></a> IoT-STH
+
+* [Introduction] (#section1)
+* [Dependencies](#section2)
+* [Installation](#section3)
+* [Inserting data (single events and aggregated data) into the database](#section4)
+* [Querying data from the database](#section5)
+* [Running the STH server](#section6)
```

Comment:
I would describe how to start the server inmmediatelly after the installation step.